File: PHP/Utils/Driver.php
<?php
use Facebook\WebDriver\Remote\RemoteWebDriver;
include('Environments.php');

function CreateDriver($desiredCapability)
{
  $driver_ = RemoteWebDriver::create(
    getLocalServer(),
    $desiredCapability,
    $connection_timeout_in_ms = 1200000,
    $request_timeout_in_ms = 1200000);
  $driver_->manage()->timeouts()->implicitlyWait(10); //seconds
  return $driver_;
}

function QuitDriver($driver){
  if($driver) {
    $driver->quit();
  }
}
?>

File: Ruby/Utils/Driver.rb
require 'selenium-webdriver'
require_relative 'Environments'

def CreateDriver cap
  driver_ = Selenium::WebDriver.for(
    :remote,
    :url => getLocalServer,
    :desired_capabilities => cap
  )
  driver_.manage.timeouts.implicit_wait = 20
  return driver_
end

def QuitDriver driver
  driver.quit
end
